neuraxle/hyperparams/distributions.py

```diff
@@ -1007,3 +1016,141 @@ def narrow_space_from_best_guess(self, best_guess, kept_space_ratio: float = 0.5
         return Normal(
             new_mean, new_std, self.hard_clip_min, self.hard_clip_max, self.null_default_value
         ).was_narrowed_from(kept_space_ratio, self)
+
+
+class DistributionMixture(HyperparameterDistribution):
```

Comment:
If you have time at the end, it would be good to update this example too to add a section at the end for the DistributionMixture, mixing a RandInt with a Uniform and a LogUniform just for the example: 
- https://www.neuraxle.org/stable/random_distributions.html

(perhaps in another PR)

Reply:
Will do in the future.